script.js

```diff
@@ -0,0 +1,8 @@
+document.querySelector('.menu-icon').addEventListener('click', () => {
+  document.querySelector('.nav-desk').classList.toggle('active');
```

Comment:
As requested by the previous reviewer, please make sure that your links redirect the user to the corresponding page 😉.
Kindly notice that currently, you have plain `li` items as `nav` links:
![image](https://user-images.githubusercontent.com/42799579/202025431-70b9b7e1-e3e7-4700-b28a-06535af65175.png)

To achieve the desired result, kindly add an `a` tag with the corresponding `href`.
You could have something like this:

```HTML
<ul>
    <li><a href="#">Hello</a></li>
    <li><a href="#portfolio">Portfolio</a></li>
    <li><a href="#about">About</a></li>
    <li><a href="#contact">Contact</a></li>
</ul>
```

If you are interested in learning more about `a` tags, check [this link](https://www.w3schools.com/tags/tag_a.asp) 😄.